Petar_Nedyalkov/task2.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+
+struct OrderPair
+{
+	int first;
+	int second;
+	void readPair()
+	{
+		std::cin >> first >> second;
+	}
+
+	void printPair()
+	{
+		std::cout << first <<" "<< second << std::endl;
+	}
+};
+
+int comparePairs(const OrderPair a, const OrderPair b)
+{
+	if (a.first > b.first || a.first == b.first && a.second > b.second) return -1; //firstpair is bigger
+	else if (a.first < b.first || a.first == b.first && a.second < b.second) return 1; //secondpair is bigger
+	else return 0; //equal 
+}
+
+OrderPair findBiggestPair(const OrderPair* arr, size_t size)
+{
+	OrderPair maxPair = arr[0];
+	for (size_t i = 1; i < size; i++)
+		if (comparePairs(arr[i], maxPair) == -1) maxPair = arr[i];
```

Comment:
You better use a comparison `< 0` as the rule of thumb in programming is `negative` for `left is smaller than right`, `zero` for `left is equal to right` and `positive` for `left is bigger than right` :))